File: src/gatling/scala/org/openapitools/client/model/GetSuggestedOfferMappingsRequest.scala

package org.openapitools.client.model


case class GetSuggestedOfferMappingsRequest (
    /* Список товаров. */
    _offers: Option[List[SuggestedOfferDTO]]
)
object GetSuggestedOfferMappingsRequest {
    def toStringBody(var_offers: Object) =
        s"""
        | {
        | "offers":$var_offers
        | }
        """.stripMargin
}

File: ympa_fsharp_functions_server/src/model/GetSuggestedOfferMappingsRequest.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.SuggestedOfferDTO

module GetSuggestedOfferMappingsRequest =

  //#region GetSuggestedOfferMappingsRequest

  [<CLIMutable>]
  type GetSuggestedOfferMappingsRequest = {
    [<JsonProperty(PropertyName = "offers")>]
    Offers : SuggestedOfferDTO[];
  }

  //#endregion
